src/clientiface.cpp

```diff
@@ -197,6 +197,9 @@ void RemoteClient::GetNextBlocks (
 	s32 nearest_sent_d = -1;
 	//bool queue_is_full = false;
 
+	v3s16 cam_pos_nodes = floatToInt(camera_pos, BS);
```

Comment:
I knew that would come up. :)
We're kinda schizophrenic about this. I'll change it.